src/NHibernate/Dialect/MsSql2000Dialect.cs

```diff
@@ -66,35 +66,412 @@ protected virtual void RegisterDefaultProperties()
 
 		protected virtual void RegisterKeywords()
 		{
-			RegisterKeyword("top");
-			RegisterKeyword("int");
-			RegisterKeyword("integer"); // a commonly-used alias for 'int'
-			RegisterKeyword("tinyint");
-			RegisterKeyword("smallint");
+			RegisterKeyword("absolute");
```

Comment:
Ok, does not really matter, leave it as many functions. It seems that C# does not bother with optimizing string arrays (as it does for numbers)